components/merkledb/tests/proof_map_index.rs

```diff
@@ -141,35 +155,41 @@ fn index_data(
     btree_map(array::uniform32(key_bytes), any::<u64>(), sizes)
 }
 
+fn data_for_absent(key_bytes: RangeInclusive<u8>) -> impl Strategy<Value = Vec<Key>> {
+    vec(array::uniform32(key_bytes), 20).prop_map(|key| vec![Key(*key.get(0).unwrap())])
```

Comment:
This transform is seemingly incorrect: why are all keys ignored except for the first one?